target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -115,6 +123,11 @@ case "$FIRMWARE" in
 		/usr/bin/fritz_cal_extract -i 1 -s 0x3D000 -e 0x212 -l 12064 -o /lib/firmware/$FIRMWARE $(find_mtd_chardev "urlader0") || \
 		/usr/bin/fritz_cal_extract -i 1 -s 0x3D000 -e 0x212 -l 12064 -o /lib/firmware/$FIRMWARE $(find_mtd_chardev "urlader1")
 		;;
+	linksys,ea8300)
+		ath10kcal_extract "0:ART" 36864 12064
+		# OEM assigns 4 sequential MACs
+		ath10kcal_patch_mac_crc $(macaddr_setbit_la $(macaddr_add "${base_mac}" 4))
```

Comment:
Is it necessary to have macaddr_setbit_la here or is it because in the stock firmware "eth0" and that wifi shares the same MAC?

As for the ${base_mac}.  
I would just follow nbg6617,wre6606 there and put the ```$(cat /sys/class/net/eth0/address)```
into the ath10kcal_patch_mac_crc parameter.

```ath10kcal_patch_mac_crc $(macaddr_add $(cat /sys/class/net/eth0/address) _X)```